src/routes/auth/user/+page.svelte

```diff
@@ -0,0 +1,70 @@
+<script lang="ts">
+  import Main from './components/Main.svelte';
+  import { firebaseDb } from '../../../lib/firebase/firebase.client';
+  import { collection, getDocs } from 'firebase/firestore';
+
+  interface Repository {
+    user_uid: string;
+    html_url: string;
+    contributors: any[];
+    languages: Record<string, any>;
+    stargazers_count: number;
+  }
+
+  interface User {
+    bio: string | null;
+    blog: string;
+    created_at: string;
+    html_url: string;
+    name: string | null;
+    public_repos: number;
+    repos_url: string;
+    total_private_repos: number;
+    username: string;
+  }
```

Comment:
There should be these types in /src/lib/types/types.ts, or if they differ it's better to move them there